styles.css

```diff
@@ -221,6 +221,34 @@ p {
   animation-duration: 0.4s;
 }
 
+.comment-display {
```

Comment:
Fix height of idea input area so inputs do not overflow when page size is reduced.